test/test_python_dispatch.py

```diff
@@ -1156,6 +1156,16 @@ def unwrap(t):
             with PoliteMode():
                 a.abs()
 
+    def test_disable_mode(self):
+        class FailEverythingMode(TorchDispatchMode):
+            def __torch_dispatch__(self, func, types, args=(), kwargs=None):
+                raise RuntimeError("arf")
+
+        with FailEverythingMode().push() as m:
```

Comment:
nit: now that #80998 is in, could we just do `with FailEverythingMode as m:`

Just trying to get in the habit of moving towards this one to avoid having 2 APIs that do the same thing

Reply:
ya, this PR predates that, I've rebased past it :)